client-telemetry-reporter-plugin/kafka/README.md

```diff
@@ -0,0 +1,163 @@
+<!-- title: How to centrally monitor Kafka clients via broker configuration -->
+<!-- description: In this tutorial, learn how to centrally monitor Kafka clients via broker configuration (using KIP-714). -->
+
+# How to centrally monitor Kafka clients via broker configuration
+
+Kafka's [KIP-714](https://cwiki.apache.org/confluence/display/KAFKA/KIP-714%3A+Client+metrics+and+observability)
+introduced in Apache Kafka 3.7 adds new capability that allows the Kafka broker to centrally track client
+metrics on behalf of applications. The broker can subsequently relay these metrics to a remote monitoring
+system, facilitating the effective monitoring of Kafka client health and the identification of any problems.
+
+The broker requires a Metrics Reporter plugin which implements the `ClientTelemetry` interface to
+send client metrics to a remote monitoring system. This tutorial demonstrates how to use the plugin
+to monitor Kafka clients.
+
+This tutorial adds `ClientOtlpMetricsReporter` that aggregates [OpenTelemetry Protocol](https://opentelemetry.io/docs/specs/otel/protocol/) (OTLP) metrics
+received from client, enhances them with additional client labels and forwards them via gRPC client
+to an external OTLP receiver. The OTLP receiver can be any OpenTelemetry compatible backend like
+Prometheus, Jaeger, Zipkin, etc. The plugin is implemented in Java and requires the jar to be
+added to the Kafka broker libs.
+
+The OTLP receiver for the tutorial is [otel-collector](https://opentelemetry.io/docs/collector/), which is a part of the OpenTelemetry project.
+The `otel-collector` receives the metrics from the Kafka broker and forwards them to a Prometheus
+instance for visualization.
+
+## Prerequisites
+
+* Java 17 or higher, e.g., follow the OpenJDK installation instructions [here](https://openjdk.org/install/) if you don't have Java.
```

Comment:
I suppose Apache Kafka 3.8.0 or later is a dependency too.

Reply:
I skipped it as prerequisite as later we ask to download and setup.